### ISSUE TITLE
Make voltage selection more user-friendly

### ISSUE THREAD

**Issue by ChucklesOnGitHub** [open-ephys/Bonsai.ONIX#13]
We talked about having a dropdown or pop-up to make voltage selection more straightforward for the users.

**Comment by jonnew** [comment]
Yeah and this is easy. Another option is to have dropdowns for cable type,
cable length, and headstage.
